task1/task1/Program.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+			Console.ReadKey();
+			Console.ForegroundColor = ConsoleColor.Green;
+			Console.WriteLine("Николай Гумилев. На горах розовеют снега...");
+			Console.ReadKey();
+			Console.ForegroundColor = ConsoleColor.Yellow;
+			Console.WriteLine("На горах розовеют снега,");
+			Console.WriteLine("Я грущу с каждым мигом сильней,");
+			Console.WriteLine("Для кого я сбирал жемчуга");
+			Console.WriteLine("В зеленеющей бездне морей ?");
+			Console.ResetColor();
+		}
```

Comment:
Старайтесь размещать закрывающую фигурную скобку под закрывающей
```suggestion
        {
			Console.ReadKey();
			Console.ForegroundColor = ConsoleColor.Green;
			Console.WriteLine("Николай Гумилев. На горах розовеют снега...");
			Console.ReadKey();
			Console.ForegroundColor = ConsoleColor.Yellow;
			Console.WriteLine("На горах розовеют снега,");
			Console.WriteLine("Я грущу с каждым мигом сильней,");
			Console.WriteLine("Для кого я сбирал жемчуга");
			Console.WriteLine("В зеленеющей бездне морей ?");
			Console.ResetColor();
        }
```